cmd/nodecmd/create.go

```diff
@@ -1488,6 +1498,29 @@ func defaultAvalancheCLIPrefix(region string) (string, error) {
 	return usr.Username + "-" + region + constants.AvalancheCLISuffix, nil
 }
 
+func sendMetrics(cmd *cobra.Command, cloudService, network string, nodes map[string]NumNodes) {
+	flags := make(map[string]string)
+	totalValidatorNodes := 0
+	totalAPINodes := 0
+	for region := range nodes {
+		totalValidatorNodes += nodes[region].numValidators
+		totalAPINodes += nodes[region].numAPI
+		flags[region] = strconv.Itoa(nodes[region].numValidators)
```

Comment:
should you also add a constant taken prefix to discriminate this is a metrics region item?